mcs_benchmark_data/_benchmark_submission_transformer.py

```diff
@@ -1,7 +1,7 @@
 import json

```

Comment:
Checked in mixed case loaded/ files. In a follow-up PR please nuke them and use lowercase names (data/mcscript/loaded/mcscript.ttl.bz2). You may need to delete the files in one commit then recreate them in another.